src/main/java/org/folio/circulation/support/Clients.java

```diff
@@ -338,6 +340,10 @@ public CollectionResourceClient noticeTemplatesClient() {
     return noticeTemplatesClient;
   }
 
+  public CollectionResourceClient actualCostRecordStorageClient() {
```

Comment:
Almost all other clients are named as [name of the entity (plural)] + "Storage" like usersStorage, loansStorage, materialTypesStorage. So it should be actualCostRecordsStorage